chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -1731,7 +1733,8 @@ async def mint_from_xch(
 
         # Aggregate everything into a single spend bundle
         total_spend = SpendBundle.aggregate([signed_spend_bundle, xch_spend, *eve_spends])
-        return total_spend
+        tx_record: TransactionRecord = dataclasses.replace(eve_txs[0], spend_bundle=total_spend)
```

Comment:
I haven't looked at the rest of this function, but are the rest of `eve_txs` worth returning too?  It's fine if they don't have a spend bundle attached, just everything that a wallet should show to a user is important.

Reply:
eve_txs comes from calling `generate_signed_transaction` so it's just one tx (in a list)